include/crm/common/acl_internal.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015-2022 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#ifndef CRM_COMMON_INTERNAL__H
+#define CRM_COMMON_INTERNAL__H
```

Comment:
Aha, here you go :)

Every header has a header guard to make sure it doesn't cause problems if it's included twice. So, it has to be unique -- this is the guard for common/internal.h, so once that's included, this won't ever be. We have a pattern for them, this one should be CRM_COMMON_ACL_INTERNAL__H.